marklogic-client-api/src/main/java/com/marklogic/client/impl/OkHttpServices.java

```diff
@@ -5782,7 +5788,23 @@ private RequestBody makeRequestBody(CallField[] params) {
                   multiBldr.addFormDataPart(paramName, null, makeRequestBody(paramValue));
                 });
           }
-        } else {
+        } else if (param instanceof BufferedMultipleNodeCallField) {
+            BufferableHandle[] paramValues = ((BufferedMultipleNodeCallField) param).getParamValuesArray();
+            if(paramValues != null) {
```

Comment:
Instead of constructing a stream, could a loop perform the same operations just as easily:

    for (BufferableHandle paramValue: paramValues) {
        // continue if null
        // same code as body of the forEachOrdered
    }